www/index.php

```diff
@@ -229,77 +230,35 @@
+				<input id ="dynamicNodeToggleCheckbox" name ="markDynamicNode" data-dojo-type="dijit/form/CheckBox" value="agreed" checked="false"/>
 				<div id="questionMarkDynamic" class="questionMark"></div>
 			</div>
 
-			<div class="fieldgroup">
-				<label for="messageBox">Messages</label>
-				<div id="messageBox" class="textscroll" data-dojo-type="dijit/layout/ContentPane"></div>
-
-				<div style="margin-bottom:10px; display:block">
-					<span>&nbsp;</span>
-					<button id="closeButton" type="button" data-dojo-type="dijit/form/Button" iconClass="dijitNoIcon" style="float:right;">Done</button>
-					<button id="deleteButton" type="button" data-dojo-type="dijit/form/Button" iconClass="dijitNoIcon" style="float:left;">Delete Node</button>
-				</div>
-			</div>
-
-		</div><!-- end of quantity node editor -->
-
-		<!-- crisis Alert -->
-		<div class="claro crisisDialog" id="crisisAlertMessage" data-dojo-type="dijit.Dialog" title="Message">
-			<div id = "crisisMessage"> </div>
-			<button id="OkButton" type="button" data-dojo-type="dijit/form/Button">OK</button>
-		</div> 
-		<!-- Equation Node Editor-->
-		<div class="claro sameedit" data-dojo-type="dijit/Dialog" id="equationNodeEditor">
-			
-			<div id="studentModelControl2" class="fieldgroup">
-				<label style="width:20ex;" for="setStudentNode2">Given to student</label>
-				<input id="setStudentNode2" name="markStudentNode2" data-dojo-type="dijit/form/CheckBox" checked="false"/>
-			</div>
-			<div id="selectModelControl2" class="fieldgroup" style="display:none" >
-				<label for="selectModel2">Select Model</label>
-				<select id="selectModel2" data-dojo-type="dijit/form/Select">
-					<option value='correct' selected>Author's Values</option>
-					<option value='given'>Initial Student Values</option>
-				</select>
-			</div>
-
-			<div class="fieldgroup">
-				<div id="descriptionControlAuthor2" class="fieldgroup" style="">
-					<span class="fixedwidth">
-						<div id="authorDescriptionQuestionMark2" class="questionMark"></div>
-						<label for="setDescription2">Explanation</label>
-					</span>
-					<input id="setDescription2" data-dojo-type="dijit/form/ComboBox">
-				</div>
-			</div>
-
-			<div class="ExpressionContainer" id="expressionDiv" style="">
+			<div class="equationInputsContainer" id="expressionDiv" style="display: none;">
 				<div class="fieldgroup">
 					 <div class="vertical">
 						<div id="equationLabel"></div>
-							<div id="nameControl2" class="fieldgroup">
-								<label for="setName2">Equation</label>
-									<textarea id="setName2" rows=4 cols=50 data-dojo-type="dijit/form/SimpleTextarea	" style="min-height:60px;"></textarea>
+							<div id="equationInputboxContainer" class="fieldgroup">
+								<label for="equationInputbox">Equation</label>
+									<textarea id="equationInputbox" rows=4 cols=50 data-dojo-type="dijit/form/SimpleTextarea" style="min-height:60px;"></textarea>
```

Comment:
I think there is some issue with indentation here? Second thing is the limit of 50 is on size of the equation or the size of the message box?